frontend/packages/console-plugin-sdk/src/codegen/__tests__/plugin-integration-tests.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { PluginPackage } from '../plugin-resolver';
+import { getPluginIntegrationTestSuites } from '../plugin-integration-tests';
+import { templatePackage } from './plugin-resolver.spec';
+
+describe('getPluginIntegrationTestSuites', () => {
+  it('will not fail on missing tests.', () => {
+    const pluginPackages: PluginPackage[] = [
+      {
+        ...templatePackage,
+        name: '@console/qux-plugin',
+        version: '2.3.4',
+        consolePlugin: { entry: 'plugin.ts' },
+      },
+    ];
+    const expectedTests = {};
+
+    expect(getPluginIntegrationTestSuites(pluginPackages)).toEqual(expectedTests);
+  });
+
+  it('returns a map of short-plugin-name to a list of its integration tests.', () => {
```

Comment:
Let's uphold the existing convention to support both full (e.g. `@console/kubevirt-plugin`) and short (e.g. `kubevirt-plugin`) plugin name from test execution (`--suite <name>`) perspective.